projects/quml-library/src/lib/mcq-option/mcq-option.component.ts

```diff
@@ -9,6 +10,7 @@ import { UtilService } from '../util-service';
 })
 export class McqOptionComponent implements OnChanges {
 
+  @Input() shuffle_options: boolean
```

Comment:
@himanshi-2602 Please use camelCase for variables

Reply:
okay